---
layout: category
title: Publications
---

## Preprints 

* A. Ozier-Lafontaine, C. Fourneaux, G. Durif, C. Vallot, O. Gandrillon, S. Giraud, B. Michel, F. Picard, Kernel-Based Testing for Single-Cell Differential Analysis, arXiv:2307.08509, 2023
* R. Belhakem, F. Picard, V. Rivoirard, and A. Roche. Minimax estimation of functional principal components from noisy discretized functional data. arXiv preprint arXiv:2110.12739, 2021
* C. Fourneaux, L. Racine, C. Koering, S. Dussurgey, E. Vallin, A. Moussy, R. Parmentier, F. Brunard, D. Stockholm, L. Modolo, et al. Differentiation is accompanied by a progressive loss in transcriptional memory. bioRxiv, 2022


## Articles

* J. Poulet-Benedetti, C. Tonnerre-Doncarli, A.-L. Valton, M. Laurent, M. Gerard, N. Barinova, N. Parisis, F. Massip, F. Picard, and M.-N. Prioleau. Dimeric G-quadruplex motifs determine a large fraction of strong replication origins in vertebrates. Nat Commun, 14, 1:4843.
* M. Lepetit, M. D. Ilie, M. Chanal, G. Raverot, P. Bertolino, F. Picard, and O. Gandrillon. scan10:A reproducible and standardized pipeline for processing 10x single cell RNAseq data. In Silico Biol, 15, 1-2:11-21.
* H. Van Assel, T. Espinasse, J. Chiquet, and F. Picard. A probabilistic graph coupling view of dimension reduction. Neurips, 2022
* A. Bonnet, C. Lacour, F. Picard, and V. Rivoirard. Uniform deconvolution for poisson point processes. Journal of Machine Learning Research, 23(194):1–36, 2022
* S. Zreika, C. Fourneaux, E. Vallin, L. Modolo, R. Seraphin, A. Moussy, E. Ventre, M. Bouvier, A. Ozier-Lafontaine, A. Bonnaffoux, et al. Evidence for close molecular proximity between reverting and undifferentiated cells. BMC biology, 20(1):1–16, 2022
* J. Käfer, A. Bewick, A. Andres-Robin, G. Lapetoule, A. Harkess, J. Caïus, B. Fogliani, G. Gâteblé, P. Ralph, C. W. dePamphilis, et al. A derived zw chromosome system in amborella trichopoda, representing the sister lineage to all other extant flowering plants. New Phytologist, 233(4):
1636–1642, 2022
* J. Käfer, N. Lartillot, G. A. B. Marais, and F. Picard. Detecting sex-linked genes using genotyped individuals sampled in natural populations. Genetics, 218(2), 03 2021b 
* J. E. Mold, L. Modolo, J. Hård, M. Zamboni, A. J. Larsson, M. Stenudd, C.-J. Eriksson, G. Durif, P. L. Ståhl, E. Borgström, S. Picelli, B. Reinius, R. Sandberg, P. Réu, C. Talavera-Lopez, B. Andersson, K. Blom, J. K. Sandberg, F. Picard, J. Michaëlsson, and J. Frisén. Divergent clonal differentiation trajectories establish cd8+ memory t cell heterogeneity during acute viral infections in humans. Cell Reports, 35(8):109174, 2021. ISSN 2211-1247
* C. Brossas, A.‐L. Valton, S. V Venev, S. Chilaka, A. Counillon, M. Laurent, C. Goncalves, B. Duriez, F. Picard, J. Dekker, M.‐N. Prioleau, Clustering of strong replicators associated with active promoters is sufficient to establish an early‐replicating domain, [The EMBO Journal (2020) e99520](https://doi.org/10.15252/embj.20189952)
* H. Badouin, A. Velt, F. Gindraud, T Flutre, V. Dumas, S. Vautrin, W. Marande, J. Corbi, E. Sallet, J. Ganofsky, S. Santoni, D. Guyot, E. Ricciardelli, K. Jepsen, J.  Käfer, Jos, H. Berges, E. Duchêne, F. Picard, P. Hugueney, R. Tavares, R. Bacilieri, C. Rustenholz, G. Marais, The wild grape genome sequence provides insights into the transition from dioecy to hermaphroditism during grape domestication, [Genome Biology 2020 (Accepted)](https://www.biorxiv.org/content/early/2020/01/10/2020.01.07.897082)
* F. Massip, M. Laurent, C. Brossas, J.-M. Fernández-Justel, M. Gómez, M.-N. Prioleau, L. Duret, F. Picard, Evolution of replication origins in vertebrate genomes: rapid turnover despite selective constraints, [Nucleic acids research 47 (10), 5114-5125](https://hal.archives-ouvertes.fr/hal-02349718)
* F. Picard, P. Reynaud-Bouret, and E. Roquain. Continuous testing for poisson process intensities: A new perspective on scanning statistics. Biometrika, 2018 [arxiv version](https://arxiv.org/abs/1705.08800)
* A. Muyle, N. Zemp, C. Fruchard, R. Cegan, J. Vrana, C. Deschamps, R. Tavares, R. Hobza, F. Picard, A. Widmer, and G. A. B. Marais. Genomic imprinting mediates dosage compensation in a young plant XY system. Nat Plants, Aug 2018 [PCI version](https://hal.archives-ouvertes.fr/hal-01851376)
* G. Durif, L. Modolo, J. E. Mold, and F. a. Lambert-Lacroix, Sophie Picard. Probabilistic count matrix factorization for single cell expression data analysis. In RECOMB, 2018 [arxiv version](https://arxiv.org/abs/1710.11028)
* G. Durif, L. Modolo, J. Michaelsson, J. Mold, S. Lambert-Lacroix, and F. Picard. High dimensional classification with combined adaptive sparse pls and logistic regression. [Bioinformatics, 2017](https://academic.oup.com/bioinformatics/article/34/3/485/4157444)
* Giacofci M, Lambert-Lacroix S, Picard F: Minimax wavelet estimation for multisample heteroscedastic nonparametric regression Journal of Nonparametric Statistics: 1–24, 2017 [arxiv version](https://arxiv.org/abs/1511.04556) 
* Durif G, Modolo L, Michaelsson J, Mold JE, Lambert-Lacroix S, Picard F: High Dimensional Classification with combined Adaptive Sparse PLS and Logistic Regression [Bioinformatics, 2017](https://academic.oup.com/bioinformatics/article/34/3/485/4157444)
* Viallon V, Lambert-Lacroix S, Hoefling H, Picard F: On the robustness of the generalized fused lasso to prior specifications Statistics and Computing 26: 285–301, 2016
* Bonafede E, Picard F, Robin S, Viroli C: Modeling overdispersion heterogeneity in differential expression analysis using mixtures Biometrics 72: 804–814, 2016
* Ivanoff S, Picard F, Rivoirard V: Adaptive Lasso and group-Lasso for functional Poisson regression Journal of Machine Learning Research 17: 1–46, 2016
* Lopez-Maestre H, Brinza L, Marchet C, Kielbassa J, Bastien S, Boutigny M, Monnin D, Filali AE, et al.: SNP calling from RNA-seq data without a reference genome: identification, quantification, differential analysis and impact on the protein sequence Nucleic acids research 44: e148–e148, 2016
* Muyle A, Käfer J, Zemp N, Mousset S, Picard F, Marais GAB: SEX-DETector: a probabilistic approach to study sex chromosomes in non-model organisms Genome biology and evolution 8: 2530–2543, 2016
* Miele V, Picard F, Dray S: Spatially constrained clustering of ecological networks Methods in Ecology and Evolution 5: 771–779, 2014
* Modolo L, Picard F, Lerat E: A new genome-wide method to track horizontally transferred sequences: application to Drosophila Genome biology and evolution 6: 416–432, 2014
* Picard F, Cadoret J-C, Audit B, Arneodo A, Alberti A, Battail C, Duret L, Prioleau M-N, et al.: The spatiotemporal program of DNA replication is associated with specific combinations of chromatin marks in human cells PLoS genetics 10: e1004282, 2014
* Giacofci M, Lambert-Lacroix S, Marot G, Picard F: Wavelet-Based Clustering for Mixed-Effects Functional Models in High Dimension Biometrics 69: 31–40, 2013
* Mestre O, Domonkos P, Picard F, Auer I, Robin S, Lebarbier E, Böhm R, Aguilar E, et al.: HOMER: a homogenization software–methods and applications, 2013
* Miele V, Penel S, Daubin V, Picard F, Kahn D, Duret L: High-quality sequence clustering guided by network topology and multiple alignment likelihood Bioinformatics 28: 1078–1085, 2012
* Vernoux T, Brunoud G, Farcot E, Morin V, Daele H Van den, Legrand J, Oliva M, Das P, et al.: The auxin signalling network translates dynamic input into robust patterning at the shoot apex Molecular systems biology 7: 508, 2011
* Picard F, Lebarbier É, Budinská E, Robin S: Joint segmentation of multivariate Gaussian processes using mixed linear models Computational Statistics & Data Analysis 55: 1160–1170, 2011
* Picard F, Lebarbier E, Hoebeke M, Rigaill G, Thiam B, Robin S: Joint segmentation, calling, and normalization of multiple CGH profiles Biostatistics 12: 413–428, 2011
* Picard F, Schbath S, Lebarbier E, Neuvial P, Chiquet J: Statistiques et génome la Gazette des Mathématiciens 130: 51–82, 2011
* Lebarbier E, Picard F: Segmentation pour l’analyse de puces CGH ESAIM: Probability and Statistics 1: 1–16, 2011
* Picard F: Statistique et génome: en guise d’introduction Gazette des Mathematiciens: 51–60, 2011
* Wiel MA Van de, Picard F, Van Wieringen WN, Ylstra B: Preprocessing and downstream analysis of microarray DNA copy number profiles Briefings in bioinformatics 12: 10–21, 2010
* Zanghi H, Picard F, Miele V, Ambroise C: Strategies for online inference of model-based clustering in large and growing networks The Annals of Applied Statistics 4: 687–714, 2010
* Picard F, Miele V, Daudin J-J, Cottret L, Robin S: Deciphering the connectivity structure of biological networks using MixNet BMC bioinformatics 10: S17, 2009
* Daudin J-J, Picard F, Robin S: A mixture model for random graphs Statistics and computing 18: 173–183, 2008
* Picard F, Daudin J-J, Koskas M, Schbath S, Robin S: Assessing the exceptionality of network motifs Journal of Computational Biology 15: 1–20, 2008
* Picard F, Robin S, Lebarbier E, Daudin J-J: A segmentation/clustering model for the analysis of array CGH data Biometrics 63: 758–766, 2007
* Guedj M, Della-Chiesa E, Picard F, Nuel G: Computing power in case-control association studies through the use of quadratic approximations: application to meta-statistics Annals of human genetics 71: 262–270, 2007
* Lubrano-Berthelier C, Dubern B, Lacorte J-M, Picard F, Shapiro A, Zhang S, Bertrais S, Hercberg S, et al.: Melanocortin 4 receptor mutations in a large cohort of severely obese adults: prevalence, functional classification, genotype-phenotype relationship, and lack of association with binge eating The Journal of Clinical Endocrinology & Metabolism 91: 1811–1818, 2006
* Picard F, Robin S, Lavielle M, Vaisse C, Daudin J-J: A statistical approach for array CGH data analysis BMC bioinformatics 6: 27, 2005
* Govaerts C, Srinivasan S, Shapiro A, Zhang S, Picard F, Clement K, Lubrano-Berthelier C, Vaisse C, et al.: Obesity-associated mutations in the melanocortin 4 receptor provide novel insights into its function Peptides 26: 1909–1919, 2005
* Srinivasan S, Lubrano-Berthelier C, Govaerts C, Picard F, Santiago P, Conklin BR, Vaisse C: Constitutive activity of the melanocortin-4 receptor is maintained by its N-terminal domain and plays a role in energy homeostasis in humans Journal of Clinical Investigation 114: 1158, 2004


