<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1" />

  <title>
    
      Publications &middot; Franck Picard
    
  </title>

  


  <!-- CSS -->
  <link rel="stylesheet" href="/assets/css/main.css" />
  

<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tajawal" />


  <!-- Icons -->
  

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/feed.xml" />

  <!-- Additional head bits without overriding original head -->
</head>


  <body class="page">

    <div id="sidebar">
  <header>
    <div class="site-title">
      <a href="/">
        
          <span class="back-arrow icon"><svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
  <path d="M0 0h24v24H0z" fill="none"/>
  <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"/>
</svg></span>
        
        Franck Picard
      </a>
    </div>
    <p class="lead"></p>
  </header>
  <nav id="sidebar-nav-links">
  
  

  

  


  
    
  

  
    
      <a class="page-link "
          href="/_pages/conferences.html">Conferences</a>
    
  

  
    
  

  

  
    
      <a class="page-link "
          href="/_pages/organization.html">Organization</a>
    
  

  
    
      <a class="page-link  active"
          href="/_pages/publications.html">Publications</a>
    
  

  
    
      <a class="page-link "
          href="/_pages/research.html">Research</a>
    
  

  
    
  

  
    
      <a class="page-link "
          href="/_pages/software.html">Software</a>
    
  

  
    
      <a class="page-link "
          href="/_pages/students.html">Students & Postdocs</a>
    
  

  

  
    
  

  
    
      <a class="page-link "
          href="/_pages/teaching.html">Teaching</a>
    
  

  


  


  
    
  

  
    
  

  
    
  

  

  
    
  

  
    
  

  
    
  

  
    
  

  
    
  

  
    
  

  

  
    
  

  
    
  

  


  <!-- Optional additional links to insert in sidebar nav -->
</nav>


  

  <nav id="sidebar-icon-links">
  
<!-- 
  <a id="subscribe-link"
     class="icon" title="Subscribe" aria-label="Subscribe"
     href="/feed.xml">
    <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
    <path d="M0 0h24v24H0z" fill="none"/>
    <circle cx="6.18" cy="17.82" r="2.18"/>
    <path d="M4 4.44v2.83c7.03 0 12.73 5.7 12.73 12.73h2.83c0-8.59-6.97-15.56-15.56-15.56zm0 5.66v2.83c3.9 0 7.07 3.17 7.07 7.07h2.83c0-5.47-4.43-9.9-9.9-9.9z"/>
</svg>
  </a>

  
  
  
  

  
    <a id="tags-link"
       class="icon"
       title="Tags" aria-label="Tags"
       href="/tags.html">
      <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
    <path d="M0 0h24v24H0z" fill="none"/>
    <path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z"/>
</svg>
    </a>
  

  
    <a id="search-link"
       class="icon"
       title="Search" aria-label="Search"
       href="/search.html">
      <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
    <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/>
    <path d="M0 0h24v24H0z" fill="none"/>
</svg>
    </a>
  
 -->
  <!-- Optional additional links to insert for icons links -->
</nav>
  @2019. all rights reserved
</div>

    <main class="container">
      <header>
  <h1 class="page-title">Publications</h1>
</header>
<div class="content">
  <h3 id="preprints">Preprints</h3>

<ol class="bibliography"><li><div class="text-justify"><span id="Massip2018evolution">Massip F, Laurent M, Brossas C, Fernández-Justel JM, Gómez Marı́a, Prioleau M-N, Duret L, Picard F, et al.: <b>Evolution of Replication Origins in Vertebrate Genomes: Rapid Turnover Despite Selective Constraints.</b> <i>bioRxiv</i>, 2018</span></div></li>
<li><div class="text-justify"><span id="bonnet2017modeling">Bonnet A, Rivoirard V, Picard F: <b>Modeling Spatial Genomic Interactions with the Hawkes model</b> <i>bioRxiv</i>: 214874, 2017</span></div></li>
<li><div class="text-justify"><span id="picard2007introduction1">Picard F: <b>An introduction to mixture models</b> <i>Statistics for Systems Biology, Research Report</i>, 2007</span></div></li>
<li><div class="text-justify"><span id="picard2007introduction2">Picard F: <b>An introduction to process segmentation</b> <i>Statistics for systems biology group</i> 6, 2007</span></div></li></ol>

<h3 id="articles">Articles</h3>

<ol class="bibliography"><li><div class="text-justify"><span id="picard2018continuous">Picard F, Reynaud-Bouret P, Roquain E: <b>Continuous testing for Poisson process intensities: a new perspective on scanning statistics</b> <i>Biometrika</i> 105: 931–944, 2018</span></div></li>
<li><div class="text-justify"><span id="muyle2018genomic">Muyle A, Zemp N, Fruchard C, Cegan R, Vrana J, Deschamps C, Tavares R, Hobza R, et al.: <b>Genomic imprinting mediates dosage compensation in a young plant XY system</b> <i>Nat Plants</i> 4: 677–680, 2018</span></div></li>
<li><div class="text-justify"><span id="durif2018probabilistic">Durif G, Modolo L, Mold JE, Lambert-Lacroix S, Picard F: <b>Probabilistic Count Matrix Factorization for Single Cell Expression Data Analysis</b> <i>Research in Computational Molecular Biology: 22nd Annual International Conference, RECOMB 2018, Paris, France, April 21-24, 2018, Proceedings</i> 10812: 254, 2018</span></div></li>
<li><div class="text-justify"><span id="giacofci2018minimax">Giacofci M, Lambert-Lacroix S, Picard F: <b>Minimax wavelet estimation for multisample heteroscedastic nonparametric regression</b> <i>Journal of Nonparametric Statistics</i> 30: 238–261, 2018</span></div></li>
<li><div class="text-justify"><span id="durif2017high">Durif G, Modolo L, Michaelsson J, Mold JE, Lambert-Lacroix S, Picard F: <b>High Dimensional Classification with combined Adaptive Sparse PLS and Logistic Regression</b> <i>Bioinformatics</i>, 2017</span></div></li>
<li><div class="text-justify"><span id="viallon2016robustness">Viallon V, Lambert-Lacroix S, Hoefling H, Picard F: <b>On the robustness of the generalized fused lasso to prior specifications</b> <i>Statistics and Computing</i> 26: 285–301, 2016</span></div></li>
<li><div class="text-justify"><span id="bonafede2016modeling">Bonafede E, Picard F, Robin S, Viroli C: <b>Modeling overdispersion heterogeneity in differential expression analysis using mixtures</b> <i>Biometrics</i> 72: 804–814, 2016</span></div></li>
<li><div class="text-justify"><span id="ivanoff2016adaptive">Ivanoff S, Picard F, Rivoirard V: <b>Adaptive Lasso and group-Lasso for functional Poisson regression</b> <i>Journal of Machine Learning Research</i> 17: 1–46, 2016</span></div></li>
<li><div class="text-justify"><span id="lopez2016snp">Lopez-Maestre H, Brinza L, Marchet C, Kielbassa J, Bastien S, Boutigny M, Monnin D, Filali AE, et al.: <b>SNP calling from RNA-seq data without a reference genome: identification, quantification, differential analysis and impact on the protein sequence</b> <i>Nucleic acids research</i> 44: e148–e148, 2016</span></div></li>
<li><div class="text-justify"><span id="muyle2016sex">Muyle A, Käfer J, Zemp N, Mousset S, Picard F, Marais GAB: <b>SEX-DETector: a probabilistic approach to study sex chromosomes in non-model organisms</b> <i>Genome biology and evolution</i> 8: 2530–2543, 2016</span></div></li>
<li><div class="text-justify"><span id="miele2014spatially">Miele V, Picard F, Dray S: <b>Spatially constrained clustering of ecological networks</b> <i>Methods in Ecology and Evolution</i> 5: 771–779, 2014</span></div></li>
<li><div class="text-justify"><span id="modolo2014new">Modolo L, Picard F, Lerat E: <b>A new genome-wide method to track horizontally transferred sequences: application to Drosophila</b> <i>Genome biology and evolution</i> 6: 416–432, 2014</span></div></li>
<li><div class="text-justify"><span id="picard2014spatiotemporal">Picard F, Cadoret J-C, Audit B, Arneodo A, Alberti A, Battail C, Duret L, Prioleau M-N, et al.: <b>The spatiotemporal program of DNA replication is associated with specific combinations of chromatin marks in human cells</b> <i>PLoS genetics</i> 10: e1004282, 2014</span></div></li>
<li><div class="text-justify"><span id="giacofci2013wavelet">Giacofci M, Lambert-Lacroix S, Marot G, Picard F: <b>Wavelet-Based Clustering for Mixed-Effects Functional Models in High Dimension</b> <i>Biometrics</i> 69: 31–40, 2013</span></div></li>
<li><div class="text-justify"><span id="mestre2013homer">Mestre O, Domonkos P, Picard F, Auer I, Robin S, Lebarbier E, Böhm R, Aguilar E, et al.: <b>HOMER: a homogenization software–methods and applications</b>, 2013</span></div></li>
<li><div class="text-justify"><span id="miele2012high">Miele V, Penel S, Daubin V, Picard F, Kahn D, Duret L: <b>High-quality sequence clustering guided by network topology and multiple alignment likelihood</b> <i>Bioinformatics</i> 28: 1078–1085, 2012</span></div></li>
<li><div class="text-justify"><span id="vernoux2011auxin">Vernoux T, Brunoud G, Farcot E, Morin V, Daele H Van den, Legrand J, Oliva M, Das P, et al.: <b>The auxin signalling network translates dynamic input into robust patterning at the shoot apex</b> <i>Molecular systems biology</i> 7: 508, 2011</span></div></li>
<li><div class="text-justify"><span id="picard2011joint">Picard F, Lebarbier É, Budinská E, Robin S: <b>Joint segmentation of multivariate Gaussian processes using mixed linear models</b> <i>Computational Statistics &amp; Data Analysis</i> 55: 1160–1170, 2011</span></div></li>
<li><div class="text-justify"><span id="picard2011joinu">Picard F, Lebarbier E, Hoebeke M, Rigaill G, Thiam B, Robin S: <b>Joint segmentation, calling, and normalization of multiple CGH profiles</b> <i>Biostatistics</i> 12: 413–428, 2011</span></div></li>
<li><div class="text-justify"><span id="picard2011statistiques">Picard F, Schbath S, Lebarbier E, Neuvial P, Chiquet J: <b>Statistiques et génome</b> <i>la Gazette des Mathématiciens</i> 130: 51–82, 2011</span></div></li>
<li><div class="text-justify"><span id="lebarbier2011segmentation">Lebarbier E, Picard F: <b>Segmentation pour l’analyse de puces CGH</b> <i>ESAIM: Probability and Statistics</i> 1: 1–16, 2011</span></div></li>
<li><div class="text-justify"><span id="picard2011statistique">Picard F: <b>Statistique et génome: en guise d’introduction</b> <i>Gazette des Mathematiciens</i>: 51–60, 2011</span></div></li>
<li><div class="text-justify"><span id="van2010preprocessing">Wiel MA Van de, Picard F, Van Wieringen WN, Ylstra B: <b>Preprocessing and downstream analysis of microarray DNA copy number profiles</b> <i>Briefings in bioinformatics</i> 12: 10–21, 2010</span></div></li>
<li><div class="text-justify"><span id="zanghi2010strategies">Zanghi H, Picard F, Miele V, Ambroise C: <b>Strategies for online inference of model-based clustering in large and growing networks</b> <i>The Annals of Applied Statistics</i> 4: 687–714, 2010</span></div></li>
<li><div class="text-justify"><span id="picard2009deciphering">Picard F, Miele V, Daudin J-J, Cottret L, Robin S: <b>Deciphering the connectivity structure of biological networks using MixNet</b> <i>BMC bioinformatics</i> 10: S17, 2009</span></div></li>
<li><div class="text-justify"><span id="daudin2008mixture">Daudin J-J, Picard F, Robin S: <b>A mixture model for random graphs</b> <i>Statistics and computing</i> 18: 173–183, 2008</span></div></li>
<li><div class="text-justify"><span id="picard2008assessing">Picard F, Daudin J-J, Koskas M, Schbath S, Robin S: <b>Assessing the exceptionality of network motifs</b> <i>Journal of Computational Biology</i> 15: 1–20, 2008</span></div></li>
<li><div class="text-justify"><span id="picard2007segmentation">Picard F, Robin S, Lebarbier E, Daudin J-J: <b>A segmentation/clustering model for the analysis of array CGH data</b> <i>Biometrics</i> 63: 758–766, 2007</span></div></li>
<li><div class="text-justify"><span id="guedj2007computing">Guedj M, Della-Chiesa E, Picard F, Nuel G: <b>Computing power in case-control association studies through the use of quadratic approximations: application to meta-statistics</b> <i>Annals of human genetics</i> 71: 262–270, 2007</span></div></li>
<li><div class="text-justify"><span id="lubrano2006melanocortin">Lubrano-Berthelier C, Dubern B, Lacorte J-M, Picard F, Shapiro A, Zhang S, Bertrais S, Hercberg S, et al.: <b>Melanocortin 4 receptor mutations in a large cohort of severely obese adults: prevalence, functional classification, genotype-phenotype relationship, and lack of association with binge eating</b> <i>The Journal of Clinical Endocrinology &amp; Metabolism</i> 91: 1811–1818, 2006</span></div></li>
<li><div class="text-justify"><span id="picard2005statistical">Picard F, Robin S, Lavielle M, Vaisse C, Daudin J-J: <b>A statistical approach for array CGH data analysis</b> <i>BMC bioinformatics</i> 6: 27, 2005</span></div></li>
<li><div class="text-justify"><span id="govaerts2005obesity">Govaerts C, Srinivasan S, Shapiro A, Zhang S, Picard F, Clement K, Lubrano-Berthelier C, Vaisse C, et al.: <b>Obesity-associated mutations in the melanocortin 4 receptor provide novel insights into its function</b> <i>Peptides</i> 26: 1909–1919, 2005</span></div></li>
<li><div class="text-justify"><span id="srinivasan2004constitutive">Srinivasan S, Lubrano-Berthelier C, Govaerts C, Picard F, Santiago P, Conklin BR, Vaisse C: <b>Constitutive activity of the melanocortin-4 receptor is maintained by its N-terminal domain and plays a role in energy homeostasis in humans</b> <i>Journal of Clinical Investigation</i> 114: 1158, 2004</span></div></li></ol>

</div>

    </main>

    <!-- Optional footer content -->

  </body>
</html>
