<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1" />

  <title>
    
      Research &middot; Franck Picard
    
  </title>

  


  <!-- CSS -->
  <link rel="stylesheet" href="/assets/css/main.css" />
  

<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tajawal" />


  <!-- Icons -->
  <!--   -->

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/feed.xml" />

  <!-- Additional head bits without overriding original head -->
</head>


  <body class="page">

    <div id="sidebar">
  <header>
    <div class="site-title">
      <a href="/">
        
          <span class="back-arrow icon"><svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
  <path d="M0 0h24v24H0z" fill="none"/>
  <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"/>
</svg></span>
        
        Franck Picard
      </a>
    </div>
    <p class="lead"></p>
  </header>
  <nav id="sidebar-nav-links">
  
  

  

  


  
    
  

  
    
      <a class="page-link "
          href="/_pages/conferences.html">Conferences</a>
    
  

  
    
  

  

  
    
      <a class="page-link "
          href="/_pages/organization.html">Organization</a>
    
  

  
    
      <a class="page-link "
          href="/_pages/publications.html">Publications</a>
    
  

  
    
      <a class="page-link  active"
          href="/_pages/research.html">Research</a>
    
  

  
    
  

  
    
      <a class="page-link "
          href="/_pages/software.html">Software</a>
    
  

  
    
      <a class="page-link "
          href="/_pages/students.html">Students & Postdocs</a>
    
  

  

  
    
  

  
    
      <a class="page-link "
          href="/_pages/teaching.html">Teaching</a>
    
  

  


  


  
    
  

  
    
  

  
    
  

  

  
    
  

  
    
  

  
    
  

  
    
  

  
    
  

  
    
  

  

  
    
  

  
    
  

  


  <!-- Optional additional links to insert in sidebar nav -->
</nav>


  

  <nav id="sidebar-icon-links">
  
<!-- 
  <a id="subscribe-link"
     class="icon" title="Subscribe" aria-label="Subscribe"
     href="/feed.xml">
    <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
    <path d="M0 0h24v24H0z" fill="none"/>
    <circle cx="6.18" cy="17.82" r="2.18"/>
    <path d="M4 4.44v2.83c7.03 0 12.73 5.7 12.73 12.73h2.83c0-8.59-6.97-15.56-15.56-15.56zm0 5.66v2.83c3.9 0 7.07 3.17 7.07 7.07h2.83c0-5.47-4.43-9.9-9.9-9.9z"/>
</svg>
  </a>

  
  
  
  

  
    <a id="tags-link"
       class="icon"
       title="Tags" aria-label="Tags"
       href="/tags.html">
      <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
    <path d="M0 0h24v24H0z" fill="none"/>
    <path d="M17.63 5.84C17.27 5.33 16.67 5 16 5L5 5.01C3.9 5.01 3 5.9 3 7v10c0 1.1.9 1.99 2 1.99L16 19c.67 0 1.27-.33 1.63-.84L22 12l-4.37-6.16z"/>
</svg>
    </a>
  

  
    <a id="search-link"
       class="icon"
       title="Search" aria-label="Search"
       href="/search.html">
      <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
    <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/>
    <path d="M0 0h24v24H0z" fill="none"/>
</svg>
    </a>
  
 -->
  <!-- Optional additional links to insert for icons links -->
</nav>
  <p>
  &copy; 2019.
  <a href="/LICENSE.md">MIT License.</a>
</p>

</div>

    <main class="container">
      <header>
  <h1 class="page-title">Research</h1>
</header>
<div class="content">
  <p>The aim of my research is to develop statistical methods dedicated to the analysis of biological data, with an emphasis on computational biology and high throughput technologies.</p>

<h1 id="current-projects">Current Projects</h1>

<h3 id="single-cell-genomics">Single Cell genomics</h3>

<p>Together with <a href="http://ki.se/en/cmb/jonas-frisens-group">J. Mold</a>, <a href="http://www.ens-lyon.fr/LBMC/laboratoire/annuaire/modolo-laurent">L. Modolo</a>, <a href="">G. Durif</a> and <a href="http://membres-timc.imag.fr/Sophie.Lambert/">S. Lambert-Lacroix</a> we develop statistical methods for the analysis of single cell data. We started by proposing a new method for PLS for classification, and we recently proposed a probabilistic version of PCA adapted to the analysis of over-dispersed counts with zero inflation. Recent developments focus on ATACSeq data analysis.</p>

<ol class="bibliography"><li><div class="text-justify"><span id="durif2017probabilistic">Durif G, Modolo L, Mold JE, Picard F, Lambert-Lacroix S: <b>Probabilistic Count Matrix Factorization for Single Cell Expression Data Analysis</b> <i>arXiv preprint arXiv:171011028</i>, 2017</span></div></li></ol>
<ol class="bibliography"><li><div class="text-justify"><span id="durif2017high">Durif G, Modolo L, Michaelsson J, Mold JE, Lambert-Lacroix S, Picard F: <b>High Dimensional Classification with combined Adaptive Sparse PLS and Logistic Regression</b> <i>Bioinformatics</i>, 2017</span></div></li></ol>

<h3 id="point-process-modeling-in-genomics">Point process modeling in genomics</h3>

<p>Recent projects focus on the spatial modeling of genomic data using point processes. With <a href="">E. Roquain</a> and <a href="">P. Reynaud-Bouret</a> we developed a testing procedure based on continuous testing to compare maps of genomic features, while controling for multiple (continuous) error rates. With <a href="">A. Bonnet</a> and  <a href="">V. Rivoirard</a> we develop Hawkes models to catch spatial interactions between genomic features.</p>

<ol class="bibliography"><li><div class="text-justify"><span id="picard2017continuous">Picard F, Reynaud-Bouret P, Roquain E: <b>Continuous testing for Poisson process intensities: A new perspective on scanning statistics</b> <i>arXiv preprint arXiv:170508800</i>, 2017</span></div></li>
<li><div class="text-justify"><span id="bonnet2017modeling">Bonnet A, Rivoirard V, Picard F: <b>Modeling Spatial Genomic Interactions with the Hawkes model</b> <i>bioRxiv</i>: 214874, 2017</span></div></li></ol>
<ol class="bibliography"></ol>

<h3 id="replication-origins-in-vertebrates">Replication origins in vertebrates</h3>

<p>Together with <a href="">M.-N. Prioleau</a> and <a href="https://lbbe.univ-lyon1.fr/-Duret-Laurent-.html">L. Duret</a> we work on the spatial program of replication at fine scales in vertebrate genomes. We characterized replication origins in humans by analyzing Oriseq data, and we determined epigenetic signatures that characterize the spatio temporal program of replication. Our method of peak detection using scan statistics can be reproduced using <a href="http://localhost:4000/assets/soft/scan-method.zip">this code</a> (with a toy example). With <a href="">F. Massip</a> We are currently working on the conservations of replication origins in vertebrates, and on the better characterization of these epigenomic signatures, especially with ATACSeq data.</p>

<ol class="bibliography"><li><div class="text-justify"><span id="picard2014spatiotemporal">Picard F, Cadoret J-C, Audit B, Arneodo A, Alberti A, Battail C, Duret L, Prioleau M-N, et al.: <b>The spatiotemporal program of DNA replication is associated with specific combinations of chromatin marks in human cells</b> <i>PLoS genetics</i> 10: e1004282, 2014</span></div></li></ol>

<h3 id="functional-data-analysis">Functional Data analysis</h3>

<p>With <a href="http://membres-timc.imag.fr/Sophie.Lambert/">S. Lambert-Lacroix</a>, <a href="http://cerim.univ-lille2.fr/pages-individuelles-chercheurs/pages-perso-guillemette-marot/guillemette-marot-briend.html">G. Marot</a>, and <a href="https://perso.univ-rennes2.fr/joyce.giacofci">J. M. Giacofci</a> we developed curve clustering models that account for an inter-individual variability throught mixed functional models. We also developed a shrinkage method for functional mixed models. More recent developments concern functional models in the Poisson case, in collaboration with <a href="https://www.ceremade.dauphine.fr/~rivoirar/">V. Rivoirard</a> and <a href="https://www.ceremade.dauphine.fr/~aroche/">A. Roche</a>.</p>

<ol class="bibliography"><li><div class="text-justify"><span id="giacofci2017minimax">Giacofci M, Lambert-Lacroix S, Picard F: <b>Minimax wavelet estimation for multisample heteroscedastic nonparametric regression</b> <i>Journal of Nonparametric Statistics</i>: 1–24, 2017</span></div></li>
<li><div class="text-justify"><span id="ivanoff2016adaptive">Ivanoff S, Picard F, Rivoirard V: <b>Adaptive Lasso and group-Lasso for functional Poisson regression</b> <i>Journal of Machine Learning Research</i> 17: 1–46, 2016</span></div></li>
<li><div class="text-justify"><span id="giacofci2013wavelet">Giacofci M, Lambert-Lacroix S, Marot G, Picard F: <b>Wavelet-Based Clustering for Mixed-Effects Functional Models in High Dimension</b> <i>Biometrics</i> 69: 31–40, 2013</span></div></li></ol>

<h3 id="population-genomics-and-sex-linked-genes">Population Genomics and sex-linked genes</h3>

<p>With <a href="https://lbbe.univ-lyon1.fr/-Marais-Gabriel-.html">G. Marais</a>, <a href="https://lbbe.univ-lyon1.fr/-Lartillot-Nicolas-.html">N. Lartillot</a>, <a href="https://lbbe.univ-lyon1.fr/-Kafer-Jos-.html">Jos Kafer</a> and <a href="">A. Muyle</a> we developed a statistical approach to infer sex-linked genes based on a controlled cross. This method is available with the sexdetector software and we are currently developing a population-based model to enrich this method.</p>

<ol class="bibliography"><li><div class="text-justify"><span id="muyle2017genomic">Muyle A, Zemp N, Fruchard C, Cegan R, Vrana J, Deschamps C, Tavares R, Picard F, et al.: <b>Genomic imprinting mediates dosage compensation in a young plant XY system</b> <i>bioRxiv</i>: 179044, 2017</span></div></li></ol>
<ol class="bibliography"><li><div class="text-justify"><span id="lopez2016snp">Lopez-Maestre H, Brinza L, Marchet C, Kielbassa J, Bastien S, Boutigny M, Monnin D, Filali AE, et al.: <b>SNP calling from RNA-seq data without a reference genome: identification, quantification, differential analysis and impact on the protein sequence</b> <i>Nucleic acids research</i> 44: e148–e148, 2016</span></div></li>
<li><div class="text-justify"><span id="muyle2016sex">Muyle A, Käfer J, Zemp N, Mousset S, Picard F, Marais GAB: <b>SEX-DETector: a probabilistic approach to study sex chromosomes in non-model organisms</b> <i>Genome biology and evolution</i> 8: 2530–2543, 2016</span></div></li></ol>

<h1 id="past-projects">Past Projects</h1>

<h3 id="cgh-microarrays-analysis-and-segmentation-models">CGH microarrays analysis and segmentation models</h3>

<p>In collaboration with <a href="https://www6.inra.fr/mia-paris/Equipes/Membres/Stephane-Robin">S. Robin</a> and <a href="https://www6.inra.fr/mia-paris/Equipes/Membres/Emilie-Lebarbier">E. Lebarbier</a> developed segmentation models for the analysis of array CGH data, an array technology that maps chromosomal aberrations. Methodological questions associated with segmentation are : finding an efficient algorithm to find the breaks by dynamic programming, and estimating the number of breaks by model selection. The purpose can also be to analyse multiple sample datasets.</p>

<ol class="bibliography"><li><div class="text-justify"><span id="picard2011joint">Picard F, Lebarbier É, Budinská E, Robin S: <b>Joint segmentation of multivariate Gaussian processes using mixed linear models</b> <i>Computational Statistics &amp; Data Analysis</i> 55: 1160–1170, 2011</span></div></li>
<li><div class="text-justify"><span id="picard2011joinu">Picard F, Lebarbier E, Hoebeke M, Rigaill G, Thiam B, Robin S: <b>Joint segmentation, calling, and normalization of multiple CGH profiles</b> <i>Biostatistics</i> 12: 413–428, 2011</span></div></li>
<li><div class="text-justify"><span id="van2010preprocessing">Wiel MA Van de, Picard F, Van Wieringen WN, Ylstra B: <b>Preprocessing and downstream analysis of microarray DNA copy number profiles</b> <i>Briefings in bioinformatics</i> 12: 10–21, 2010</span></div></li>
<li><div class="text-justify"><span id="picard2007segmentation">Picard F, Robin S, Lebarbier E, Daudin J-J: <b>A segmentation/clustering model for the analysis of array CGH data</b> <i>Biometrics</i> 63: 758–766, 2007</span></div></li>
<li><div class="text-justify"><span id="picard2005statistical">Picard F, Robin S, Lavielle M, Vaisse C, Daudin J-J: <b>A statistical approach for array CGH data analysis</b> <i>BMC bioinformatics</i> 6: 27, 2005</span></div></li></ol>

<h3 id="biological-networks-random-graphs-and-motifs">Biological networks, random graphs and motifs</h3>

<p>To study the topology of complex networks we (with <a href="https://www6.inra.fr/mia-paris/Equipes/Membres/Stephane-Robin">S. Robin</a> and <a href="https://www6.inra.fr/mia-paris/Equipes/Membres/Jean-Jacques-Daudin">J.-J. Daudin</a>) proposed the Mixnet model, also known as the Stochastic Block Model, to cluster nodes of a network that share the same connectivity via a generalization of mixture models to network data. As a result, MixNet provides a synthetic picture of the main connectivity patterns that make the network. A challenge when using these models lies in the variational algorithm to estimate the parameters. We proposed an online version of such algorithm to deal with very large networks (with <a href="">C. Ambroise</a> and <a href="">V. Miele</a>).</p>

<p>Another way to study the architecture of networks is to study their basic subunits, building blocks or network motifs. The statistical question associated with network motifs is the question of their exceptionality. Are there motifs that occur more than expected in biological network ? This question is related to the distribution of the count of some subgraphs in random graphs.</p>

<ol class="bibliography"><li><div class="text-justify"><span id="viallon2016robustness">Viallon V, Lambert-Lacroix S, Hoefling H, Picard F: <b>On the robustness of the generalized fused lasso to prior specifications</b> <i>Statistics and Computing</i> 26: 285–301, 2016</span></div></li>
<li><div class="text-justify"><span id="miele2014spatially">Miele V, Picard F, Dray S: <b>Spatially constrained clustering of ecological networks</b> <i>Methods in Ecology and Evolution</i> 5: 771–779, 2014</span></div></li>
<li><div class="text-justify"><span id="miele2012high">Miele V, Penel S, Daubin V, Picard F, Kahn D, Duret L: <b>High-quality sequence clustering guided by network topology and multiple alignment likelihood</b> <i>Bioinformatics</i> 28: 1078–1085, 2012</span></div></li>
<li><div class="text-justify"><span id="vernoux2011auxin">Vernoux T, Brunoud G, Farcot E, Morin V, Daele H Van den, Legrand J, Oliva M, Das P, et al.: <b>The auxin signalling network translates dynamic input into robust patterning at the shoot apex</b> <i>Molecular systems biology</i> 7: 508, 2011</span></div></li>
<li><div class="text-justify"><span id="zanghi2010strategies">Zanghi H, Picard F, Miele V, Ambroise C: <b>Strategies for online inference of model-based clustering in large and growing networks</b> <i>The Annals of Applied Statistics</i> 4: 687–714, 2010</span></div></li>
<li><div class="text-justify"><span id="picard2009deciphering">Picard F, Miele V, Daudin J-J, Cottret L, Robin S: <b>Deciphering the connectivity structure of biological networks using MixNet</b> <i>BMC bioinformatics</i> 10: S17, 2009</span></div></li>
<li><div class="text-justify"><span id="daudin2008mixture">Daudin J-J, Picard F, Robin S: <b>A mixture model for random graphs</b> <i>Statistics and computing</i> 18: 173–183, 2008</span></div></li>
<li><div class="text-justify"><span id="picard2008assessing">Picard F, Daudin J-J, Koskas M, Schbath S, Robin S: <b>Assessing the exceptionality of network motifs</b> <i>Journal of Computational Biology</i> 15: 1–20, 2008</span></div></li></ol>

</div>

    </main>

    <!-- Optional footer content -->

  </body>
</html>
